pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/InTransformFunction.java

```diff
@@ -57,8 +57,8 @@ public String getName() {
   @Override
   public void init(List<TransformFunction> arguments, Map<String, DataSource> dataSourceMap) {
     int numArguments = arguments.size();
-    Preconditions.checkArgument(numArguments >= 2,
-        "At least 2 arguments are required for IN transform function: expression, values");
+    Preconditions.checkArgument(numArguments >= 2, "At least 2 arguments are required for ["
+        + getName() + "] transform function: expression, values");
```

Comment:
Same for other changes
```suggestion
    Preconditions.checkArgument(numArguments >= 2, "At least 2 arguments are required for [%s] transform function: expression, values", getName());
```